app/Services/PatientVerificationCodeService.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Services;
+
+use App\Repositories\PatientVerificationCodeRepository;
+use App\Repositories\SpecializationRepository;
+
+class PatientVerificationCodeService extends BaseService
+{
+    public function __construct(PatientVerificationCodeRepository $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function deletePatientOldCode($patient)
+    {
+        $code = $this->repository->fiterPatientCode($patient)->toArray();
+        if ($code) {
+           $this->delete($this->show($code[0]['id'])); 
+        }
```

Comment:
Use th approach I've commented in the repo and delete this

Reply:
done